spec/Section 6 -- Execution.md

```diff
@@ -357,8 +456,11 @@ If during {ExecuteSelectionSet()} a field with a non-null {fieldType} raises a
 _field error_ then that error must propagate to this entire selection set,
 either resolving to {null} if allowed or further propagated to a parent field.
 
-If this occurs, any sibling fields which have not yet executed or have not yet
-yielded a value may be cancelled to avoid unnecessary work.
+If this occurs, any defer or stream executions with a path that starts with the
```

Comment:
This is really "starts with but is not equal to" or something, right? A literal reading here would cause one to assume that the actual defer itself (whose path starts with itself) should not be returned to the client.

Also "same path as the resolved null" — this is a bit confusing as to whether it means the original null or a bubbled-up null.

Maybe both issues can be fixed while being clear that the paths you're comparing are the root paths of each defer/stream?

Reply:
I updated this again. Hopefully it's clearer now.

Reply:
I also added examples for all of the non-null cases

Reply:
I have now also added the complete algorithm for FilterSubsequentPayloads as implemented in https://github.com/graphql/graphql-js/pull/3720